src/scales/scale.linearbase.js

```diff
@@ -25,7 +25,7 @@ function generateTicks(generationOptions, dataRange) {
 	var niceMax = Math.ceil(dataRange.max / spacing) * spacing;
 
 	// If min, max and stepSize is set and they make an evenly spaced scale use it.
-	if (generationOptions.min && generationOptions.max && generationOptions.stepSize) {
+	if (!helpers.isNullOrUndef(generationOptions.min) && !helpers.isNullOrUndef(generationOptions.max) && !helpers.isNullOrUndef(generationOptions.stepSize)) {
```

Comment:
@teroman I just realized that we should **not** apply the same logic to `stepSize` because `stepSize === 0` is invalid (we divide by `stepSize` line 30).

Should be:
```javascript
if (!helpers.isNullOrUndef(generationOptions.min) && !helpers.isNullOrUndef(generationOptions.max) && generationOptions.stepSize) {
```